### ISSUE TITLE
Add aria-live to dynamic elements

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#647]
Parts of the interface which are updated dynamically should have an `aria-live` attribute.

These include:

- [ ] part feedback
- [ ] warning messages
- [ ] modal dialogs
